EIPS/eip-5437.md

```diff
@@ -76,7 +76,9 @@ This table can be extended by future EIPs.
 in the table above.
 
 An example of `publicKey` using `RSA/3072` (TODO: decide whether to use secp256k1 instead)
-generated via [GnuPG](https://gnupg.org/),
+generated via GnuPG
+<!-- TODO waiting link to be allowed
+https://gnupg.org/ -->
```

Comment:
This link will not be allowed

Reply:
Updated to "add proper link to openstandard"